packages/typescript/src/components/ImportStatement.tsx

```diff
@@ -21,9 +21,14 @@ export interface ImportStatementsProps {
 export function ImportStatements(props: ImportStatementsProps) {
   const pkg = usePackage();
 
+  const imports = [...props.records];
```

Comment:
I'm surprised this works given that this set is not reactive. I think that suggests that this entire component is getting re-rendered whenever an import is added?